src/components/Sidebar/SidebarHeader.tsx

```diff
@@ -156,11 +154,11 @@ const SidebarHeader: FC<SidebarHeaderProps> = ({ node, isTransition = false }) =
               className={clsx(
                 styles.headerLabel,
                 styles.headerLabelText,
-                node.type === 'start' && styles.headerLabelStart
+                node.key === 'Start' && styles.headerLabelStart
```

Comment:
why check against the `key` instead of the `type` like it was before?  `type` seems safer since its type is an enumerated list of strings instead of just a string.

Reply:
I think this was a rebase issue. Something else funky happened with these which I caught but I missed this one!